modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -3,56 +3,211 @@
+=== Local/Guest Users
 
-== Setup Within the OpenID Provider
+A solution to authenticate local/guest users via OIDC will be available soon.
+
+== Installation
+
+=== Configure ownCloud
+
+The first thing to do is to install {openid-app-marketplace-url}[the OIDC app].
+After it is installed, add the configuration below in either `config/config.php` or a custom config.php file.
+
+[source,php]
+----
+include::{examplesdir}configuration/user/oidc/config-with-service-discovery.php[]
+----
 
-When registering ownCloud as and OpenID Client, use `https://cloud.example.net/index.php/apps/openidconnect/redirect` as the redirect URL.
-In case {openid-connect-frontchannel-logout-url}[OpenID Connect Front-Channel Logout 1.0] is supported, please enter `https://cloud.example.net/index.php/apps/openidconnect/logout` as the logout URL within the client registration of the OpenID Provider.
+In the example above, the values for `provider-url`, `client-id` and `client-secret` need to be taken from the OpenId Provider setup, which we will cover shortly.
+The value for `loginButtonName` can be chosen at your discretion.
 
-== Setup Service Discovery
+The above configuration assumes that the OpenId Provider is supporting xef:setup-service-discovery[service discovery].
+If the OpenId Provider _does not_ support service discovery, the endpoint configuration has to be done manually, as in the following example.
+
+[source,php]
+----
+include::{examplesdir}configuration/user/oidc/config-without-service-discovery.php[]
+----
+
+TIP: Please refer to xref:configuration/server/config_sample_php_parameters.adoc#oidc-configuration[the OIDC configuration example] in the sample configuration documentation for more informatio on how to configure ownCloud to work with OIDC.
+
+=== Setup Within the OpenID Provider
+
+When registering ownCloud as an OpenID client, use the following URLs:
+
+[cols="2"]
+|===
+|**The redirect URL**
+|`https://cloud.example.net/index.php/apps/openidconnect/redirect`
+|**The logout URL** +
+//(within the client registration of the OpenID Provider).
+Only use this if {openid-connect-frontchannel-logout-url}[OpenID Connect Front-Channel Logout 1.0] is supported.
+|`https://cloud.example.net/index.php/apps/openidconnect/logout` 
+|===
+
+=== Setup Service Discovery
 
 To allow other clients to use OpenID Connect when talking to ownCloud, please set up a redirect on the webserver to point `.well-known/openid-configuration` to `/index.php/apps/openidconnect/config`.
-Here is an .htaccess example
+Here is an example of how to do so in .htaccess.
 
 [source]
 ----
 RewriteRule ^\.well-known/openid-configuration /index.php/apps/openidconnect/config [R=301,L]
 ----
 
-//== Integration with different IdPs
-// (e.g., Ping Identity / Kopano Konnect / Keycloak)
+TIP: Please refer to https://httpd.apache.org/docs/current/mod/mod_rewrite.html[the mod_rewrite manual] for further details on creating rewrite rules.
```

Comment:
Better use of {...link...} instead http://

Reply:
Thanks for spotting that.